src/coverlet.core/Coverage.cs

```diff
@@ -294,9 +294,10 @@ private string GetSourceLinkUrl(Dictionary<string, string> sourceLinkDocuments,
                 string key = sourceLinkDocument.Key;
                 if (Path.GetFileName(key) != "*") continue;
 
-                string relativePath = Path.GetRelativePath(Path.GetDirectoryName(key), Path.GetDirectoryName(document));
+                if (!Path.GetDirectoryName(document).StartsWith(Path.GetDirectoryName(key) + Path.DirectorySeparatorChar))
+                    continue;
 
-                if (relativePath.Contains("..")) continue;
+                var relativePath = Path.GetDirectoryName(document).Substring(Path.GetDirectoryName(key).Length + 1);
```

Comment:
If this is indeed a functional workaround to using `Path.GetRelativePath` then we can make `coverlet.msbuild` be fully compliant with `netcoreapp2.0` and `net472` frameworks by making it target `netstandard2.0` instead

Reply:
To my knowledge Coverlet worked quite well with `net472` when the target was `netstandard2.0`. Only got issues when it was changed to `netcoreapp2.0`

Reply:
It is even working with net35

Reply:
@sharwell are you still able to this? If not, I'll just merge it in and make the necessary changes if that's alright with you

Reply:
Thanks for your help so far

Reply:
@tonerdo I wanted to merge this one like this, then update and merge #301, and only then start to focus on enabling .NET Framework scenarios. Currently testing for the latter is completely blocked so I can't say for sure what will or will not work.

Reply:
I'd prefer if it gets done in this PR. #301 might take a bit to get through and I want the next release (which I want to happen this week) to fix the .NET Framework issue

Reply:
I'm not sure I'll have time to make and test this modification.

Reply:
No worries. I'll go ahead and merge this and effect the necessary changes. Thank you so much for all your work on this